src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawningPanelAdapter.java

```diff
@@ -8,11 +8,12 @@
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawningPanelAdapter implements Job2dDriver {
```

Comment:
The name does not indicate the purpose of the adapter or what was actually adapted.